packages/core/content-type-builder/admin/src/components/FormModal/FormModal.tsx

```diff
@@ -276,29 +261,38 @@ export const FormModal = () => {
         }
 
         if (modalType === 'customField') {
-          dispatch({
-            type: SET_CUSTOM_FIELD_DATA_SCHEMA,
-            customField,
-            isEditing: actionType === 'edit',
-            modifiedDataToSetForEditing: attributeToEdit,
-            // NOTE: forTarget is used in the i18n middleware
-            forTarget,
-          });
+          if (actionType === 'edit') {
+            dispatch(
+              actions.setCustomFieldDataSchema({
+                isEditing: true,
+                modifiedDataToSetForEditing: attributeToEdit,
+              })
+            );
+          } else {
+            dispatch(
+              actions.setCustomFieldDataSchema({
+                customField: customField!,
+                isEditing: false,
+                modifiedDataToSetForEditing: attributeToEdit,
+              })
+            );
+          }
         } else {
-          dispatch({
-            type: SET_ATTRIBUTE_DATA_SCHEMA,
-            attributeType,
-            nameToSetForRelation: get(collectionTypesForRelation, ['0', 'title'], 'error'),
-            targetUid: get(collectionTypesForRelation, ['0', 'uid'], 'error'),
-            isEditing: actionType === 'edit',
-            modifiedDataToSetForEditing: attributeToEdit,
-            step,
-            forTarget,
-          });
+          dispatch(
+            actions.setAttributeDataSchema({
+              attributeType,
+              nameToSetForRelation: get(collectionTypesForRelation, ['0', 'title'], 'error'),
+              targetUid: get(collectionTypesForRelation, ['0', 'uid'], 'error'),
+              isEditing: actionType === 'edit',
+              modifiedDataToSetForEditing: attributeToEdit,
+              step,
+              // forTarget,
```

Comment:
Is it intentional to leave this out?

Reply:
Yup as I went through the actions this param didn't exist. I just want to keep it commented as I continue the following refactor to keep in mind it was there before in case I find a new use for it that was lost :)